oak-segment-tar/src/main/java/org/apache/jackrabbit/oak/segment/file/tar/TarReader.java

```diff
@@ -602,22 +566,12 @@ public void close() throws IOException {
     }
 
     /**
-     * Loads and parses the optional pre-compiled graph entry from the given tar
-     * file.
+     * Loads and parses the optional pre-compiled graph entry from the given tar file.
```

Comment:
I think this should now also mention that the graph is computed if not present in the archive.